### ISSUE TITLE
Calculate box size by people

### ISSUE THREAD

**Issue by rspeeruchicago** [rspeeruchicago/Maot-Chitum#15]
Calculate a recommended box size(s) based on people in house.  JoEllyn to provide formula.  For example, 4 people get a box#2.

**Comment by rspeeruchicago** [comment]
Now that we have the logic, programming goes back to me.  Changing Assignee.

**Comment by Jstoliar** [comment]
Do I do that?


Joellyn Stoliar
Executive Director
Maot Chitim of Greater Chicago
3710 Commercial Ave, Suite 7
Northbrook, IL 60062
(847) 674-3224
www.maotchitim.org<http://www.maotchitim.org>

**Comment by rspeeruchicago** [comment]
Nope, I changed the task back to me.  It was just sending you an update because you were the owner.  You are good on that one.

Thanks,

Rob

Rob Speer
Principal Consultant
312-480-0538
[cid:image002.png@01D44C34.938A6D00]

**Comment by Jstoliar** [comment]
Thanks!

Joellyn Stoliar
Executive Director
Maot Chitim of Greater Chicago
3710 Commercial Ave, Ste 7
Northbrook, IL 60062
www.maotchitim.org
________________________________
